src/plugins/recordTypes/audit/columns.js

```diff
@@ -0,0 +1,35 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    formatTimestamp,
+  } = configContext.formatHelpers;
+
+  return {
+    default: {
+      principal: {
+        messages: defineMessages({
+          label: {
+            id: 'column.audit.default.updatedBy',
+            defaultMessage: 'By',
+          },
+        }),
+        order: 10,
+        sortBy: 'audits_common:principal',
```

Comment:
The REST API doesn't seem to support sorting audit records, so this can be removed for now. If we decide that these need to be sortable, I think we'll have to do some work in the services layer.

Reply:
(There's also a bug that's causing an error in the UI when you click on the header, but that's unrelated. I tried sending various requests to the REST API to try to get it to sort, but none of them worked).

Reply:
(And I think it should be `audit_common`, not `audits_common`, but that doesn't make it work.)